### DISCUSSION TITLE
Improved Select for Belongs To fields - Fuzzy search

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#574] [Feature requests]
Currently, the `belongs_to` field is just a dropdown displaying the titles of all the resources that the user has permission to see. In my app, this could be thousands of options. It would be great if we could have a fuzzy search of some kind to make the selection less tedious.

**Comment by adrianthedev** [comment]
Implemented by https://github.com/avo-hq/avo/pull/646
